integration/commands_administration_test.go

```diff
@@ -185,18 +185,69 @@ func TestCommandsAdministrationGetParameter(t *testing.T) {
 		databaseName: "admin",
 	})
 
-	var actual bson.D
-	err := collection.Database().RunCommand(ctx, bson.D{{"getParameter", "*"}}).Decode(&actual)
-	require.NoError(t, err)
+	for name, tc := range map[string]struct {
+		command    bson.D
+		expected   map[string]any
+		unexpected []string
+		err        *mongo.CommandError
+	}{
+		"AllParameters": {
+			command: bson.D{{"getParameter", "*"}},
```

Comment:
Let's add a test for `bson.D{{"getParameter", "*"}, {"quiet", 1}, {"quiet_other", 1}}`

Reply:
fix